communities/microgalaxy/lab/base.yml

```diff
@@ -17,14 +17,20 @@ galaxy_base_url: https://microgalaxy.usegalaxy.eu
 subdomain: microGalaxy
 root_domain: usegalaxy.eu
 support: https://help.galaxyproject.org/
+general_support: https://galaxyproject.org/support/
 
 # This will enable a feedback form on the webpage:
 # feedback_email: help@mygalaxy.org
 
-support_url: https://help.galaxyproject.org
 quota_request_url: https://help.galaxyproject.org
 data_policy_url: https://usegalaxy.org/static/terms.html
 terms_url: https://usegalaxy.org/static/terms.html
+microgalaxy_matrix: https://matrix.to/#/#galaxyproject_microGalaxy:gitter.im
+galaxy_help_url: https://help.galaxyproject.org
+gtn_slack_url: https://join.slack.com/t/gtnsmrgsbord/shared_invite/zt-2llyx6p8j-LmpEIsJu0t4MQkBctkN8qg
+gtn_matrix_channel: https://matrix.to/#/#galaxyproject_microGalaxy:gitter.im
+american_server: https://matrix.to/#/#galaxyproject_Lobby:gitter.im
+european_server: https://matrix.to/#/#usegalaxy-eu_Lobby:gitter.im
```

Comment:
These 2 links should be named sth like `server_matrix_channel` and be in corresponding server `yml` file